google/cloud/cloudquotas/v1/.repo-metadata.json

```diff
@@ -0,0 +1,14 @@
+{
+    "api_id": "cloudquotas.googleapis.com",
+    "api_shortname": "cloudquotas",
+    "client_documentation": "https://cloud.google.com/cpp/docs/reference/cloudquotas/latest",
+    "distribution_name": "google-cloud-cpp",
+    "issue_tracker": "https://issuetracker.google.com/issues?q=componentid:445904%20status=open",
+    "language": "cpp",
+    "library_type": "GAPIC_AUTO",
+    "name_pretty": "Cloud Quotas API",
+    "product_documentation": "https://cloud.google.com/docs/quota/api-overview",
```

Comment:
This still says "preview", but I suspect that is a bug in the documentation site.

Reply:
No action required.